py3status/modules/deadbeef.py

```diff
@@ -67,56 +79,48 @@ def _is_running(self):
 
     def deadbeef(self):
         color = self.color_stopped
-        artist = isplaying = length = playback_time = title = \
-            tracknumber = year = ""
+        status = ''
```

Comment:
Not necessary about truncation. Easier to find `plist` instead of `placeholders` because there are 12 `placeholders` in this file when using Ctrl+F. Same with others. We open files, then Ctrl+F `some_parameter` or use Goto LineNumber. I won't change them now though. Good to know.